watertap/property_models/IX_prop_pack.py

```diff
@@ -0,0 +1,1110 @@
+            mutable=True,
+            default=18e-3,
+            initialize=self.config.mw_data,
+            units=pyunits.kg / pyunits.mol,
+            doc="Molecular weight",
+        )
+
+        self.diffus_phase_comp = Param(
+            self.phase_list,
+            self.ion_set,
+            mutable=True,
+            default=1e-9,
+            initialize=self.config.diffusivity_data,
+            units=pyunits.m**2 * pyunits.s**-1,
+            doc="Bulk diffusivity of ion",
+        )
+
+        self.visc_d_phase = Param(
+            self.phase_list,
+            # mutable=True,
+            default=1e-3,
+            initialize=1e-3,  # TODO:revisit- assuming ~ 1e-3 Pa*s for pure water
+            units=pyunits.Pa * pyunits.s,
+            doc="Fluid viscosity",
+        )
+
+        # Ion charge
+        self.charge_comp = Param(
+            self.ion_set,
+            mutable=True,
+            default=1,
+            initialize=self.config.charge,
+            units=pyunits.dimensionless,
+            doc="Ion charge",
+        )
+
+        # traditional parameters are the only Vars currently on the block and should be fixed
+        for v in self.component_objects(Var):
+            v.fix()
+
+        # ---default scaling---
+        self.set_default_scaling("temperature", 1e-2)
+        self.set_default_scaling("pressure", 1e-6)
+        self.set_default_scaling("dens_mass_phase", 1e-3, index="Liq")
+        self.set_default_scaling("visc_d_phase", 1e3, index="Liq")
+        self.set_default_scaling("diffus_phase_comp", 1e10, index="Liq")
+        self.set_default_scaling("visc_k_phase", 1e6, index="Liq")
+
+    @classmethod
+    def define_metadata(cls, obj):
+        """Define properties supported and units."""
+        obj.add_properties(
+            {
+                "flow_mol_phase_comp": {"method": None},
+                "temperature": {"method": None},
+                "pressure": {"method": None},
+                "flow_mass_phase_comp": {"method": "_flow_mass_phase_comp"},
+                "diffus_phase_comp": {"method": "_diffus_phase_comp"},
+                # "flow_mol_phase_comp": {"method": "_flow_mol_phase_comp"},
+                "flow_equiv_phase_comp": {"method": "_flow_equiv_phase_comp"},
+                "mass_frac_phase_comp": {"method": "_mass_frac_phase_comp"},
+                "dens_mass_phase": {"method": "_dens_mass_phase"},
+                "flow_vol_phase": {"method": "_flow_vol_phase"},
+                "flow_vol": {"method": "_flow_vol"},
+                "conc_mass_phase_comp": {"method": "_conc_mass_phase_comp"},
+                "conc_mol_phase_comp": {"method": "_conc_mol_phase_comp"},
+                "conc_equiv_phase_comp": {"method": "_conc_equiv_phase_comp"},
+                # "mole_frac_phase_comp": {"method": "_mole_frac_phase_comp"},
+                "visc_d_phase": {"method": "_visc_d_phase"},
+                "visc_k_phase": {"method": "_visc_k_phase"},
```

Comment:
and ``flow_equiv_phase_comp`` -- I am wondering if it is a good idea to add a new pyunit ``equiv``? Right now both use ``mols`` because there is no equivalent pyunit as part of the standard set.